assets/src/apps/app.tsx

```diff
@@ -11,13 +10,11 @@ import { Maybe } from 'tsmonad';
  * @param name A good debug-name to see in the react-dev-tools
  * @returns The wrapped component
  */
-const wrapWithRedux = (Component: FC, name: string): FC => {
-  const store = configureStore();
```

Comment:
Instead of creating a new store for every component that is wrapped, the store is explicitly passed in